src/TraceEvent/Symbols/SymbolReader.cs

```diff
@@ -880,8 +880,10 @@ internal bool GetPhysicalFileFromServer(string serverPath, string pdbIndexPath,
                             m_log.WriteLine("FindSymbolFilePath: In task, sending HTTP request {0}", fullUri);
 
                             var req = (System.Net.HttpWebRequest)System.Net.HttpWebRequest.Create(fullUri);
-                            req.UserAgent = "Microsoft-Symbol-Server/6.13.0009.1140";
```

Comment:
Was this intentional to remove the 'UserAgent' property set?   If so we need to do a bit more testing.   At one point at least symbol servers did not reply properly if this user agent field was not set to something it recognized (hopefully they just dropped that).

Reply:
Yes. That isn't supported in 1.6. I'll go ahead and test that

Reply:
Note an easy way of doing this is to build PerfView with this line removed, remove the %TEMP%\symbols directory and then resolve some native OS Dlls from a trace.   If that works I am happy.

Reply:
It works without setting user agent. I captured the [network log](https://pastebin.com/4uzUqdUg) just to confirm